app/src/main/res/values/strings.xml

```diff
@@ -71,6 +71,7 @@
     <string name="main_launch_fragment">Launch Chat in Fragment</string>
     <string name="main_start_in_integrator_app">Start Chat in integrator app</string>
     <string name="main_start_chat_flow">Start new chat flow</string>
+    <string name="main_start_message_center_flow">Start new chat flow</string>
```

Comment:
To my mind, this name may be confusing with the current chat flow.
Maybe "Start new messaging flow" or any other name?

Reply:
done